src/Core/Usecase/HXL/SearchHXLTagAttributes.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: rowasc
+ * Date: 5/7/18
+ * Time: 2:37 PM
+ */
+
+namespace Ushahidi\Core\Usecase\HXL;
+
+use Ushahidi\Core\Usecase\SearchUsecase;
+
+class SearchHXLTagAttributes extends SearchUsecase
+{
+	// Usecase
+	public function interact()
+	{
+		// Fetch an empty entity...
+		$entity = $this->getEntity();
```

Comment:
I think this is the same as regular `searchusecase` interact and could go away

Reply:
Yea the entire class can go away if thats the case